ui/src/app/cohort-search/node-info/node-info.component.ts

```diff
@@ -198,19 +205,27 @@ export class NodeInfoComponent implements OnInit, OnDestroy, AfterViewInit {
     }
   }
 
-  showCount() {
+  get showCount() {
     return this.node.get('selectable') && this.node.get('count') !== null;
   }
 
-  isPM() {
+  get isPM() {
     return this.node.get('type') === TreeType[TreeType.PM];
   }
 
-  isDrug() {
+  get isDrug() {
     return this.node.get('type') === TreeType[TreeType.DRUG];
   }
 
-  isMeas() {
+  get isMeas() {
```

Comment:
Would it make sense to create a function(consolidate isPM(), isDrug(), isMeas()) and some variables like so:
```
private pm = TreeType[TreeType.PM];
private drug = TreeType[TreeType.DRUG];
private meas = TreeType[TreeType.MEAS];
private icd9 = TreeType[TreeType.ICD9];
private icd10 = TreeType[TreeType.ICD10];
```

```
get isNodeType(... args) {
     return args.includes(this.node.get('type'));
}
```

And you can call it like so: 
`this.isNodeType(pm); `
or 
`this.isNodeType(drug, icd9, icd10);`

Reply:
Also, should we move this into a utils component since it seems to be a re-occurring theme in many components?

Reply:
Yes, I think that's a good idea. Can do a follow-up pr soon to address here and other components as well